async-inference/Async-Inference-Walkthrough-SageMaker-Python-SDK.ipynb

```diff
@@ -0,0 +1,652 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    model_name = resource_name.format("Model")
</pre><p>What is the purpose of doing the <code>.format("Model")</code> and <code>.format("Endpoint")</code> on a plain string? Should it have braces for string substitution? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3270/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Done